torch/csrc/jit/codegen/fuser/cpu/fused_kernel.cpp

```diff
@@ -113,59 +113,60 @@ void activate() {
   rtrim(path);
 
   // Checking whether the activation script `vcvarsall.bat` exists
-  path += "\\VC\\Auxiliary\\Build";
-  struct stat st;
-  if (stat(path.c_str(), &st) == -1 || !(st.st_mode & _S_IFDIR)) {
+  path += L"\\VC\\Auxiliary\\Build";
+  struct _stati64 st;
```

Comment:
Strictly speaking, `_stat64` should be used here. See https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?view=msvc-160.